src/Containers/Popup.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { LabelLayoutProps } from '@Layouts';
+
+export interface PopupProps extends LabelLayoutProps {
+    left?: number;
+    top?: number;
+    width?: number;
+    height?: number;
+    popup?: boolean;
+}
+
+export const Popup: React.FC<PopupProps> = ({
+    children,
+    left,
+    top,
+    width,
+    height,
+    popup,
+}): React.ReactElement => {
+    return (
+        <Container popup={popup} left={left} top={top}>
+            <Rectangle width={width} height={height}>
+                {children}
+            </Rectangle>
+            <Triangle>
+                <InnerTriangle> </InnerTriangle>
```

Comment:
might want to change it to a self closed tag if not being enclosed with child/data